homework

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
```

Comment:
you need to create a Ball class with the following fields: color and number and override the toString() method
create Lottery class with method
To test the program, in the main() method, create three balls using the getRandomBall() method from the Lottery class. And print information about them in the console.